scripts/cpu-setonix.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash --login
+
+# SLURM directives
+#
+# Here we specify to SLURM we want an OpenMP job with 32 threads
+# a wall-clock time limit of one hour.
+#
+# Replace [your-project] with the appropriate project name
+# following --account (e.g., --account=project123).
+
+#SBATCH --account=pawsey0807
+#SBATCH --partition=work
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=128
+#SBATCH --cpus-per-task=1
+#SBATCH --mem=58880M      #Indicate the amount of memory per node when asking for share resources
```

Comment:
We should just request exclusive use of the node to avoid needing to do this.

Reply:
how do you do that

Reply:
SLURM cheat sheet: https://slurm.schedmd.com/pdfs/summary.pdf

Reply:
`--exclusive`